pipeline.yml

```diff
@@ -16,18 +16,25 @@ schedule :
 build:
   -
     scripts:
-      - "dotnet build --configuration Release IronPython3Extension.sln"
+      - dotnet build --configuration Release IronPython3Extension.sln
 
 ci_test:
   tests:    
     -
       scripts:
-        - "dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit"   
+        - dotnet test --no-build --configuration Release IronPython3Extension.sln --filter \"TestCategory!=Failure\" --logger:junit  --logger:trx" -f xml -o "coverage.xml"
```

Comment:
The " after trx looks wrong to me.

Reply:
And there is no need to escape " for the filters now.

Reply:
Oops missed the cover tool. We want to follow this pattern:
`dotnet coverage collect "dotnet test --no-build --configuration Release TestingSolution.sln --logger:junit --logger:trx" -f xml -o "coverage.xml"`
so the escape is still needed for the filters, etc.

Reply:
Ups, it was my bad, done!